packages/dev/docs/pages/releases/2022-11-15.mdx

```diff
@@ -0,0 +1,296 @@
+
+### Design updates
+Spectrum has also updated the designs for Button, ActionButton, ActionGroup, and ToggleButton  with improved contrast and accessibility. Button includes new outline and fill options, as well as new static color options. ActionButton, ActionGroup, ToggleButton, Picker, TextField, and other field components now have darker borders and selected states for improved contrast.
+
+The Button API has been updated to add new options, and deprecate some old ones. Deprecated options will be automatically mapped to new variants.
+
+- Added `accent` variant, replacing `cta` which is now deprecated.
+- Added `style` prop, accepting `fill` or `outline` for all button variants.
+- Added `staticColor` prop, which replaces the deprecated `overBackground` variant.
+- Deprecated `isQuiet`.
+
+View the updates on our [Button docs](https://react-spectrum.adobe.com/react-spectrum/Button.html#visual-options) and [Spectrum Design docs](https://spectrum.corp.adobe.com/page/button/#Accent-variant).
+
+## Fixes
+- Avoid widening "type" prop of `TextInputDOMProps` - [@jossmac](https://github.com/jossmac) - [PR](https://github.com/adobe/react-spectrum/pull/3623)
+- Expose `props` to `Section` collection item - [@luisadame](https://github.com/luisadame) - [PR](https://github.com/adobe/react-spectrum/pull/3634)
+- Clear slots for `ContextualHelp` slot provider for `Dialog` - [@jluyau](https://github.com/jluyau) - [PR](https://github.com/adobe/react-spectrum/pull/3635)
+- Fix focus for empty view in `TableView` and `ListView` - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3581)
+- Fix `TableView` crashing when adding/removing columns - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/3656)
+- Fix `DatePicker` and `DateField` key/focus event props - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3605)
+- Export `FocusVisibleProps` - [@warsawgentleman](https://github.com/warsawgentleman) - [PR](https://github.com/adobe/react-spectrum/pull/3639)
+- Remove unneeded aria-checked from `useCheckbox` and `useSwitch` - [@tywayne](https://github.com/tywayne) - [PR](https://github.com/adobe/react-spectrum/pull/3687)
+- Fix dragging in `ListView` with selectionMode set to `none` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3696)
+- Add generic for `DatePickerStateOptions` - [@jason89521](https://github.com/jason89521) - [PR](https://github.com/adobe/react-spectrum/pull/3682)
+- Update date components to use `useFocusWithin` - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3694)
```

Comment:
Minor suggestion, it might be clearer to say "Update date components events to use".